README.md

```diff
@@ -23,21 +23,21 @@ You can then use Chronos:
 <?php
 require 'vendor/autoload.php';
 
-use Cake\Chronos\Chronos;
+use Cake\Chronos\DateTime;
 
-printf("Now: %s", Chronos::now());
+printf("Now: %s", DateTime::now());
 ```
```

Comment:
A fair concern.
Technically it's impossible to import both without aliasing one.
And if you use Chronos you would want to use it consistently across the code base.
Doing it half/half brings you only in trouble.

Apart from that, there should never be the use case of both in the same class file, unless upgrading partially from legacy code or alike. In those cases one can use the Chronos alias or use inline `\DateTime` maybe?

In general, nobody prevents you from the alias completely of course if that makes you feel more comfortable I guess.